Automattic/pocket-casts-android
# Migrate Trial Finished to Compose UI

## Description
- This is the first PR to migrate the Trial Finished screen to compose UI. I will make some improvements on scroll in another PR
- Created the `GradientIcon` component 


## Testing Instructions
1. Apply [trial.patch](https://github.com/user-attachments/files/17951296/trial.patch)
2. See the screen. You can try with different devices

## Screenshots or Screencast 

<img width="1649" alt="image" src="https://github.com/user-attachments/assets/dc0c47cc-7191-418b-95b6-98fe57d91c39">

<img width="1664" alt="image" src="https://github.com/user-attachments/assets/f72e2e45-973d-4694-94dd-9fcc150d5a52">


### Small Device Experience


[small device.webm](https://github.com/user-attachments/assets/eca6cba7-9f18-4d4e-8d1a-8239960e96c4)



## Checklist
 
#### I have tested any UI changes...
